src/Files.App/Actions/Navigation/PreviousTabAction.cs

```diff
@@ -26,14 +29,15 @@ public PreviousTabAction()
 			multitaskingContext.PropertyChanged += MultitaskingContext_PropertyChanged;
 		}
 
-		public Task ExecuteAsync(object? parameter = null)
+		public async Task ExecuteAsync(object? parameter = null)
 		{
 			if (App.AppModel.TabStripSelectedIndex is 0)
 				App.AppModel.TabStripSelectedIndex = multitaskingContext.TabCount - 1;
 			else
 				App.AppModel.TabStripSelectedIndex--;
 
-			return Task.CompletedTask;
+			await Task.Delay(500);
```

Comment:
Can you add a comment for future reference?

Reply:
Done